msdsn/conn_str.go

```diff
@@ -569,6 +571,14 @@ func Parse(dsn string) (Config, error) {
 		p.Encoding.GuidConversion = false
 	}
 
+	disableEPA, ok := params[DisableEPA]
+	if ok {
```

Comment:
>if ok { [](http://example.com/codeflow?start=1&length=7)

perhaps for the first release the default behavior should be the same as before and only use the EPA code if `disableEPA` is added to the connection string explicitly as `false` or `0`. It could also use an env var like `MSSQL_USE_EPA` to control the behavior for folks that don't want to have to update their connection strings.